pkg/workload/schemachange/schemachange.go

```diff
@@ -1086,6 +1091,36 @@ func (w *schemaChangeWorker) randType(tx *pgx.Tx) (tree.ResolvableTypeReference,
 	return rowenc.RandSortingType(w.rng), nil
 }
 
+func (w *schemaChangeWorker) createSchema(tx *pgx.Tx) (string, error) {
+	schemaName, err := w.randSchema(tx, 10)
+	if err != nil {
+		return "", err
+	}
+
+	stmt := rowenc.MakeSchemaName(w.rng.Intn(2) == 0, schemaName, "root")
```

Comment:
It may be better to leave authorization plank until the workload supports multiple roles.